### PR TITLE
docs: update readme

### PR DESCRIPTION
**Pull request by bbssyyuui**
update readme

### CODE REVIEW

**Comment by ChasonTang** [comment]
Look good to me.

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+.DS_Store
```
Comment (ChasonTang) [inline .gitignore:1]: Use github/gitignore macOS.gitignore instead

**Review by ChasonTang — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (bbssyyuui) [reply .gitignore]: 修改已完成

Comment (ChasonTang) [reply .gitignore]: ok

**Review by ChasonTang — APPROVED** [review]
Look good to me.

**Pull request merged**